wpiunits/src/main/java/edu/wpi/first/units/Per.java

```diff
@@ -109,6 +109,15 @@ public D denominator() {
     return m_denominator;
   }
 
+  /**
+   * Returns the reciprocal of this Per.
+   *
+   * @return the reciprocal
+   */
+  public Per<D, N> reciprocal() {
+    return m_denominator.per(m_numerator);
+  }
+
```

Comment:
Should adding this be a seperate pr?

Reply:
It's helpful for simplifying the dimensional analysis in the divide function. It makes sense to keep it as part of this PR